test/cli/test/anypattern_skip_error/policy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: validate-service-loadbalancer
+spec:
+  validationFailureAction: enforce
+  rules:
+  - name: check-loadbalancer-public
+    match:
+      resources:
```

Comment:
Please use an `any` or `all` block under the `match` clause. We have deprecated not doing so and, going forward, this will be a requirement.

Reply:
Done.